common/lib/xmodule/xmodule/tests/test_randomize_module.py

```diff
@@ -97,8 +97,8 @@ def inner_get_module(descriptor):
         randomize_module = RandomizeModule(
             randomize_descriptor,
             self.system,
-            DictFieldData({}),
-            ScopeIds(None, None, self.course.id, self.course.id)
+            field_data=None,
```

Comment:
If you don't pass `field_data` here, then the block will get it from the runtime (`self.system`, in this case).